docs/partials/deploy-hyperlane/_deploy-contracts.mdx

```diff
@@ -42,6 +42,33 @@ hyperlane config create ism
 
 When asked about the multisig type, choose `message id ism`. In the context of this guide, we will be using a 1/1 multisig, so choose a threshold of `1` and enter the address of your key.
 
+### Dry Run
+
+Prior to executing a deployment, you may now execute a dry-run via `--dry-run`, `-dr`, or `-d` to ensure that the deployment will be successful.
+This will not deploy any contracts, but you will be able to see all the contracts that will be deployed, along with the generated artifacts at `$OUT_DIR/dry-run_*`.
+
+In addition to validating your upcoming deployment, you can provide the address of the EOA you intend to deploy with via `--key` or `-k`.
+This will ensure you have approximately enough funds to complete the deployment, without providing access to your private key to pay for transactions.
+Additionally, gas usage statistics will be displayed in your console to provide insight into the cost of deployment.
+
+:::note
+
+In order to execute a dry-run, you will need an Anvil node running in a separate terminal instance.
+To spin up an Anvil node, run `anvil`.
+For more on Anvil and installation, checkout [Foundry's Anvil docs](https://book.getfoundry.sh/anvil/).
```

Comment:
does the CLI assume localhost 8545?
any way to configure this?

Reply:
Yes it does. I'll include info on how to set once we enable multi-network support for dry-running